Android/app/src/main/java/io/github/project_travel_mate/friend/FriendCityRecyclerAdapter.java

```diff
@@ -0,0 +1,62 @@
+package io.github.project_travel_mate.friend;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import io.github.project_travel_mate.R;
+import objects.FriendCity;
+
+public class FriendCityRecyclerAdapter extends RecyclerView.Adapter<FriendCityRecyclerAdapter.FriendCityViewHolder> {
+
+    private ArrayList<FriendCity> mCityList;
+    private Context mContext;
+
+    public FriendCityRecyclerAdapter(Context context, ArrayList<FriendCity> list) {
+        mCityList = list;
+        mContext = context;
+    }
+
+    @NonNull
+    @Override
+    public FriendCityViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        return new FriendCityViewHolder(
+                LayoutInflater.from(mContext).inflate(R.layout.friend_city_list_item, parent, false));
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull FriendCityViewHolder holder, int position) {
+        holder.friendCityTextView.setText(mCityList.get(position).getCityName());
+        holder.friendCityTextView.setBackgroundColor(getRandomColor());
```

Comment:
remove this also. Let it be orange color only...

Reply:
Done. 👍